vendor/github.com/openshift/library-go/pkg/operator/resource/resourceread/config.go

```diff
@@ -1,58 +0,0 @@
-package resourceread
```

Comment:
Why this file got deleted? There are no changes to go.mod and I couldn't find any refs in this PR to `{Read,Write}FeatureGate*` functions, so if we don't use these functions why this file was not cleaned rebase or other changes?

Reply:
This is indeed weird. A `make vendor` restored the file.